### ISSUE TITLE
How to add new games

### ISSUE THREAD

**Issue by sneccc** [moonlessformless/ps2_cam_acolyte#1]
Hello, can you do a video tutorial explaining how you go about creating a freecam scrip for a particular game?

**Comment by moonlessformless** [comment]
Yes - I'll try to do this in the next few days and will post a VOD here to close the issue. I'd also like to write up a .md file with the basics as well.

**Comment by SixelAlexiS90** [comment]
> Yes - I'll try to do this in the next few days and will post a VOD here to close the issue. I'd also like to write up a .md file with the basics as well.

Thanks! It would be really cool using it on ICO and SoTC, looking forward to it :'D

**Comment by moonlessformless** [comment]
Still haven't had a chance to stream or record a VOD, but I did write up this hacking guide:
https://github.com/moonlessformless/ps2_cam_acolyte/blob/main/HACKING_GUIDE.md
